src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square implements Figure {
```

Comment:
The class `Square` should not start with an empty line. According to the checklist, you should remove all redundant empty lines at the beginning of class or method implementations.